tests/utils_tests/test_database_url.py

```diff
@@ -0,0 +1,257 @@
+# -*- coding: utf-8 -*-
+# !/usr/bin/env python
+
+import unittest
+
+from django.db import connection
+from django.utils.url_config import configure_db
+
+
+@unittest.skipUnless(connection.vendor == 'postgres', 'Postgres tests')
```

Comment:
The downside of this approach is that the DB module needs to be imported, which will fail unless the DB-specific adapters are installed (`psycopg2` etc).